packages/core/src/user-store/postgres/store.ts

```diff
@@ -61,6 +60,23 @@ export class PostgresUserStore implements UserStore {
     });
   }
 
+  linkToCurrent = async () => {
+    if (!this.schema) return;
+
+    await this.db.transaction().execute(async (tx) => {
+      await Promise.all(
+        this.schema!.entities.map(async (model) => {
+          const tableName = `${model.name}_${this.versionId}`;
+          tx.schema
+            .createView(model.name)
+            .orReplace()
```

Comment:
Replacing a view breaks if the schema changes, e.g. we added a new column to the table. It's safer to delete a view and recreate them instead of replacing.